OpenApi execution of openapi resulted in exception: NullPointerException

Hi, I use **oneOf** Parameter but maven execute NullPointerException
`Failed to execute goal io.jooby:jooby-maven-plugin:2.8.9:openapi (default) on project  : execution of openapi resulted in exception: NullPointerException -> [Help 1]`
```
@ApiResponse(responseCode = "200",
      content = {
          @Content(mediaType = MediaType.JSON, schema = @Schema(type = "object",oneOf = {Result.class}))
          })
```

thx~   

stacktrace as follow
```
Caused by: org.apache.maven.plugin.MojoFailureException: execution of openapi resulted in exception
    at io.jooby.maven.BaseMojo.execute (BaseMojo.java:83)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at org.codehaus.classworlds.Launcher.main (Launcher.java:47)
Caused by: java.lang.NullPointerException
    at io.jooby.internal.openapi.OpenAPIParser.toSchema (OpenAPIParser.java:593)
    at io.jooby.internal.openapi.OpenAPIParser.arrayOrSchema (OpenAPIParser.java:557)
    at io.jooby.internal.openapi.OpenAPIParser.mediaType (OpenAPIParser.java:543)
    at io.jooby.internal.openapi.OpenAPIParser.lambda$content$70 (OpenAPIParser.java:522)
    at io.jooby.internal.openapi.AsmUtils.annotationList (AsmUtils.java:108)
    at io.jooby.internal.openapi.OpenAPIParser.content (OpenAPIParser.java:520)
    at io.jooby.internal.openapi.OpenAPIParser.operationResponse (OpenAPIParser.java:511)
    at io.jooby.internal.openapi.OpenAPIParser.lambda$null$65 (OpenAPIParser.java:480)
    at java.util.ArrayList.forEach (ArrayList.java:1257)
    at io.jooby.internal.openapi.OpenAPIParser.lambda$responses$66 (OpenAPIParser.java:480)
    at io.jooby.internal.openapi.AsmUtils.annotationList (AsmUtils.java:108)
    at io.jooby.internal.openapi.OpenAPIParser.responses (OpenAPIParser.java:479)
    at io.jooby.internal.openapi.OpenAPIParser.operation (OpenAPIParser.java:375)
    at io.jooby.internal.openapi.OpenAPIParser.lambda$parse$28 (OpenAPIParser.java:197)
    at java.util.Optional.ifPresent (Optional.java:159)
    at io.jooby.internal.openapi.OpenAPIParser.parse (OpenAPIParser.java:197)
    at io.jooby.internal.openapi.RouteParser.parse (RouteParser.java:66)
    at io.jooby.openapi.OpenAPIGenerator.generate (OpenAPIGenerator.java:165)
    at io.jooby.maven.OpenAPIMojo.doExecute (OpenAPIMojo.java:60)
    at io.jooby.maven.BaseMojo.execute (BaseMojo.java:78)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at org.codehaus.classworlds.Launcher.main (Launcher.java:47)

--- a/modules/jooby-openapi/src/main/java/io/jooby/internal/openapi/OpenAPIParser.java
+++ b/modules/jooby-openapi/src/main/java/io/jooby/internal/openapi/OpenAPIParser.java
@@ -604,7 +604,7 @@ public class OpenAPIParser {
 
     List<io.swagger.v3.oas.models.media.Schema> schemas = schemaMap.get("implementation");
     io.swagger.v3.oas.models.media.Schema schema;
-    if (schemas.isEmpty()) {
+    if (schemas == null || schemas.isEmpty()) {
       ComposedSchema composedSchema = new ComposedSchema();
 
       Optional.ofNullable(schemaMap.get("anyOf")).ifPresent(composedSchema::anyOf);

--- a/modules/jooby-openapi/src/test/java/issues/i1855/App1855.java
+++ b/modules/jooby-openapi/src/test/java/issues/i1855/App1855.java
@@ -1,0 +1,9 @@
+package issues.i1855;
+
+import io.jooby.Jooby;
+
+public class App1855 extends Jooby {
+  {
+    mvc(new Controller1855());
+  }
+}

--- a/modules/jooby-openapi/src/test/java/issues/i1855/Controller1855.java
+++ b/modules/jooby-openapi/src/test/java/issues/i1855/Controller1855.java
@@ -1,0 +1,22 @@
+package issues.i1855;
+
+import examples.Person;
+import io.jooby.MediaType;
+import io.jooby.annotations.GET;
+import io.jooby.annotations.Path;
+import io.jooby.annotations.PathParam;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+
+@Path("/openapi")
+public class Controller1855 {
+
+  @GET("/{id}")
+  @ApiResponse(responseCode = "200", content = {
+      @Content(mediaType = MediaType.JSON, schema = @Schema(type = "object", oneOf = { Person.class }))
+  })
+  public Person get(@PathParam long id) {
+    return null;
+  }
+}

--- a/modules/jooby-openapi/src/test/java/issues/i1855/Issue1855.java
+++ b/modules/jooby-openapi/src/test/java/issues/i1855/Issue1855.java
@@ -1,0 +1,20 @@
+package issues.i1855;
+
+import io.jooby.internal.openapi.ResponseExt;
+import io.jooby.openapi.OpenAPITest;
+import io.jooby.openapi.RouteIterator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class Issue1855 {
+
+  @OpenAPITest(value = App1855.class, ignoreArguments = true)
+  public void shouldParseSwaggerAnnotations(RouteIterator iterator) {
+    iterator.next(route -> {
+          ResponseExt response = route.getDefaultResponse();
+          assertEquals("200", response.getCode());
+          assertEquals("Success", response.getDescription());
+          assertEquals("examples.Person", response.getJavaType());
+        }).verify();
+  }
+}

